doc/design.md

```diff
@@ -157,7 +157,7 @@ Returns: auth token
 - `400 BAD FORMAT`: missing email or password
 
 
-**`GET`** `/api/users/`
+### **`GET`** `/api/users/`
```

Comment:
need to add in the response:
- name
- group

Reply:
Fixed! thanks.